ethereum/forge-test/query/QueryTest.sol

```diff
@@ -120,6 +120,80 @@ library QueryTest {
             _accounts
         );
     }
+        
+    /// @dev buildSolanaPdaRequestBytes builds a sol_pda query request from the specified fields.
+    function buildSolanaPdaRequestBytes(
+        bytes memory _commitment,
+        uint64 _minContextSlot,
+        uint64 _dataSliceOffset,
+        uint64 _dataSliceLength,
+        uint8 _numPdas,
+        bytes memory _pdas // Created with buildSolanaPdaEntry()
+    ) internal pure returns (bytes memory){
+        return abi.encodePacked(
+            uint32(_commitment.length),
+            _commitment,
+            _minContextSlot,
+            _dataSliceOffset,            
+            _dataSliceLength,
+            _numPdas,
+            _pdas
+        );
+    }
+
+    /// @dev buildSolanaPdaEntry builds a PDA entry for a sol_pda query.
+    function buildSolanaPdaEntry(
```

Comment:
Does it make sense to have an "entries" variant that calls this method to build an array of Pda entries?

Reply:
I looked into doing this but it got pretty messy (having to create an array of structs which would contain an array). Instead, I changed `buildSolanaPdaRequestBytes` to take an array of PDAs and updated the test to show how it can be done pretty easily.